ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/private/cucushift-installer-rehearse-gcp-ipi-private-workflow.yaml

```diff
@@ -0,0 +1,13 @@
+workflow:
+  as: cucushift-installer-rehearse-gcp-ipi-private
+  steps: 
+    pre:
+    - chain: ipi-gcp-private-pre
```

Comment:
We encourage to use flatten ref/chain steps in install workflow yaml definition instead of a wrapper chain for multiple steps, that will help be easier for troubleshooting, also help get understanding which steps are referenced in this workflow. It is better to keep the same style as other platforms.